Docs/Configuration.tex

```diff
@@ -5966,6 +5966,19 @@ \subsection{Output Properties}\label{uefioutputprops}
   the firmware does not provide it, it can be added with \texttt{ProvideConsoleGop}
   set to \texttt{true}.
 
+\item
+  \texttt{ForceResolution}\\
+  \textbf{Type}: \texttt{plist\ boolean}\\
+  \textbf{Failsafe}: \texttt{false}\\
+  \textbf{Description}: Forces \texttt{Resolution} to be set in cases where the desired
+  resolution is not available by default, such as on legacy Intel GMA and first
+  generation Intel HD Graphics (Ironlake/Arrandale). Setting \texttt{Resolution} to
+  \texttt{Max} will try to pull the largest available resolution from the connected
+  display's EDID.
+
+  \emph{Note}: This option depends on the \href{https://github.com/acidanthera/OpenCorePkg/blob/master/Include/Acidanthera/Protocol/OcForceResolution.h}{\texttt{OC\_FORCE\_RESOLUTION\_PROTOCOL}}
+  protocol being present. This protocol is currently only supported by \texttt{OpenDuetPkg}.
```

Comment:
Should probably add for Intel GPUs since in theory at least AMD GPUs may also need that.